### DISCUSSION TITLE
NFC plugin : application crashes on launch.

### DISCUSSION THREAD

**Discussion by mokmap** [driftyco/ionic-native#4451] [Q&A]
I use Ionic 6 capacitor and I want to use NFC. I have installed the NFC plugin but it has not been updated for 2 years. Therefore it does not work. 

Could someone help me? I can't find an alternative to this plugin.

**Comment by seikosantana** [comment]
Try using `phonegap-nfc-api31` instead of `phonegap-nfc`
